python/init.sls

```diff
@@ -1,12 +1,24 @@
+{% set python_version = pillar.get('python_version', '2.7') ~ '' %}
```

Comment:
It is and it also does a type conversion so if someone had their pillar set as
```
python_version: 2.7
```
vs 
```
python_version: '2.7'
```
then the later comparison would still work. Perhaps there is a better way to do that...